.gitignore

```diff
@@ -40,3 +40,10 @@ language/tools/vm-genesis/genesis/vm_config.toml
 
 # Terraform
 .terraform/
+
+# VS Code
+.vscode/
```

Comment:
there was a discussion about not adding user-specific gitignores to the repo, this might help: https://stackoverflow.com/questions/25912441/can-i-add-a-gitignore-file-just-for-me-that-overrides-the-project-gitignore

@bmwill might have other suggestions.